instant-xml/src/lib.rs

```diff
@@ -185,7 +185,7 @@ impl<'xml, W: std::fmt::Write> Serializer<'xml, W> {
     }
 
     pub fn consume_field_context(&mut self) -> Option<FieldContext<'xml>> {
-        let ret = self.next_field_contest.clone();
+        let ret = self.next_field_contest.take();
```

Comment:
You can just do `self.next_field_contest.take()` (by the way, should rename that `contest` -> `context`), it will leave the field as `None`.

Reply:
done